Makefile

```diff
@@ -92,33 +92,36 @@ install-protoc:
 install-compile-proto: install-protoc compile-proto
 
 install-controller-gen:
-	# Build controller-gen from go.mod
```

Comment:
Could you please remove `tidy` target too?

Reply:
I dont get.